src/libra_py/workflows/pyxaid2/namd.py

```diff
@@ -286,6 +286,64 @@ def sac_matrices(coeff):
     return P2C
 
 
+def scale_H_vib(hvib, dE, dNAC, sc_nac_method):
+    """
+    This function scales the energies and NACs in the vibrionic Hamiltonian
+    in the Chi basis.   
+    hvib = CMATRIXlist of vibronic hamiltonians in the Chi basis
+    dE   = The desired energy gap (E_1 - E_0), for the Chi basis
+    dNAC = The scaling terms by which specific nacs will be scaled
```

Comment:
that's not convenient to user - they would need to first set up the whole list of lists of lists object to do the scaling. Should be simple - just a matrix of numbers - constant for all times. If you want to keep the present version (more flexible), then add another function that would do the constant factor scaling by first creating the scaling object and then passing it to this function.